<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UvDialog</class>
 <widget class="QDialog" name="UvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UV-VIS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 255, 255);
}

/*checkbox字体和间距设置*/
QCheckBox{
	spacing: 10px;
	font: 12pt &quot;黑体&quot;;
}

/*checkbox样式设置*/
QCheckBox::indicator { 
	width: 64px;
	height: 32px;
}
/*未选中*/
QCheckBox::indicator::unchecked {	
	image: url(:/icons/uncheck1.png);
}
/*选中*/
QCheckBox::indicator::checked { 
	image: url(:/icons/check1.png);
}

QPushButton,QLabel,QLineEdit,QSpinBox,QDoubleSpinBox{
	min-height:48px;
	font: 12pt &quot;黑体&quot;;
}

QGroupBox{
	font: Bold 14pt &quot;黑体&quot;;
}

/*spinbox 抬起样式*/
QTimeEdit::up-button,QDoubleSpinBox::up-button,QSpinBox::up-button {subcontrol-origin:border;
    subcontrol-position:right;
	image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;       
}
QTimeEdit::down-button,QDoubleSpinBox::down-button,QSpinBox::down-button {subcontrol-origin:border;
    subcontrol-position:left;
	image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 36px;
    height: 50px;
}

/*按钮按下样式*/
QTimeEdit::up-button:pressed,QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icons/up_left.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;       
}
  
QTimeEdit::down-button:pressed,QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
    subcontrol-position:left;
    image: url(:/icons/up_right.png);
	background-color: rgb(255, 255, 255);
    width: 35px;
    height: 49px;   
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,1">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QDialog{
	background-color: rgb(255, 255, 255);
}

/*checkbox字体和间距设置*/
QCheckBox{
	spacing: 10px;
	font: 12pt &quot;黑体&quot;;
}

/*checkbox样式设置*/
QCheckBox::indicator { 
	width: 64px;
	height: 32px;
}
/*未选中*/
QCheckBox::indicator::unchecked {	
	image: url(:/icons/uncheck1.png);
}
/*选中*/
QCheckBox::indicator::checked { 
	image: url(:/icons/check1.png);
}

QPushButton,QLabel,QLineEdit,QSpinBox{
	min-height:48px;
	font: 12pt &quot;黑体&quot;;
}

QGroupBox{
	font: Bold 14pt &quot;黑体&quot;;
}
</string>
     </property>
     <property name="title">
      <string>Watch</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="wl1CheckBox">
          <property name="text">
           <string>Wave Length1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wl1SpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> nm</string>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>850</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Coef1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cf1DoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="wl2CheckBox">
          <property name="text">
           <string>Wave Length2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wl2SpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> nm</string>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>850</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Coef2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cf2DoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="wl3CheckBox">
          <property name="text">
           <string>Wave Length3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wl3SpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> nm</string>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>850</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Coef3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cf3DoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="wl4CheckBox">
          <property name="text">
           <string>Wave Length4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wl4SpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> nm</string>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>850</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Coef4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cf4DoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="singleStep">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="wl5CheckBox">
          <property name="text">
           <string>Wave Length5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wl5SpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> nm</string>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>850</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Coef5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cf5DoubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelPBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okPBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
